src/routes/volunteer.router.ts

```diff
@@ -9,5 +11,7 @@ const volunteerRouter = express.Router();
 /* Admin Controller */
 volunteerRouter.post('/', createVolunteer);
 volunteerRouter.get('/allVolunteers', getVolunteers);
+volunteerRouter.patch('/:volunteerId/approve', approveVolunteer);
+volunteerRouter.patch('/:volunteerId/deny', denyVolunteer);
```

Comment:
I can see the argument of why this makes more sense having them separate, intuitively it does look better but for the sake of the frontend code we'll save quite a few lines if we can coming them. Good job using patch!